### PR TITLE
Add user current program to mongo

### PR DESCRIPTION
**Pull request by xinhaoz**

### CODE REVIEW

**Comment by GodGreg** [comment]
Thank you for doing this! I'll give this a thorough review this weekend.

File: src/Components/Admin/ProgramSelect.js [hunk 1]
```diff
@@ -11,17 +11,9 @@ const Wrapper = styled.div`
...
-  { name: "SVP Teens", id: "SVP Teens" },
-  { name: "SVP Perfect Pitch", id: "SVP Perfect Pitch" }
-];
-
 function ProgramSelect({ value, onChange }) {
-
-  const [programs] = usePromise(GET.getAllProgramsAPI, {}, [])
-  console.log(programs)
+  const [programs] = usePromise(GET.getAllProgramsAPI, {}, []);
+  console.log(programs);
```
Comment (ahmedhamodi) [inline src/Components/Admin/ProgramSelect.js:16]: Can remove this console log.

**Review by ahmedhamodi — APPROVED** [review]
Looks good to me code wise. If you can resolve the merge conflict and remove that one console log, I think it's good!

File: src/Components/StackedRankings/StackedRankings.jsx [hunk 2]
```diff
@@ -148,6 +151,10 @@ function StackedRankings({ applications, reviewCount, user }) {
             rank["Organization Name (legal name)"] || rank["Organization Name"]
         }))
       );
+      UPDATE.updateStackedAPI({
+        userId: user.userId,
+        rankings: reorderedList.map((app) => ({ appId: app._id }))
```
Comment (ahmedhamodi) [inline src/Components/StackedRankings/StackedRankings.jsx:156]: This reorderedList variable is not defined earlier in this file.

**Pull request merged**